package.json

```diff
@@ -79,35 +87,31 @@
     ],
     "devDependencies": {
         "@babel/core": "^7.6.4",
-        "@babel/polyfill": "^7.6.0",
+        "@babel/plugin-proposal-export-default-from": "^7.5.2",
```

Comment:
> I think using a stage-1 feature in a high profile project as web3.js is a mistake.

I think this is exactly the great thing about those plugins. We can use already stage1 or stageX proposals without having to care about the support. The used stage1 proposal can also get removed and will manually write down what the plugin would do.

> At the very least, a change like this should be introduced in a PR of its own and properly debated.

True that. This was the reason for the other PR and I thought it's easier to review anything if all the changes are in one place. :)

Reply:
> The used stage1 proposal can also get removed and I will manually write down what the plugin would do.

The risk is that underestimating this work is very easy. 

Proposals in stages < 3 do change. As an example, TS implemented decorators before that, and now their implementation doesn't match the proposal's spec. I've also seen examples of this happening to application developers, not just lang/tooling ones.

This particular proposal isn't a big deal, because it's pretty small. But adding those kind of plugins is a slippery slope.

Reply:
> This particular proposal isn't a big deal, because it's pretty small. But adding those kind of plugins is a slippery slope.

True that 👌 
I think this is an allowed exception and is also not used extensively (only modules with named exports do use them).